CMakeLists.txt

```
@@ -20,10 +20,11 @@ set_target_properties(LDtkLoader PROPERTIES
 )
 
 # examples targets
-option(LDTK_NO_THROW      "Replace all throws by an error print and exit" OFF)
-option(BUILD_SFML_EXAMPLE "Build an example using LDtkLoader with SFML." OFF)
-option(BUILD_SDL_EXAMPLE  "Build an example using LDtkLoader with SDL."  OFF)
-option(BUILD_API_TEST     "Build an example that loads a file showcasing all features of LDtk."  OFF)
+option(LDTK_NO_THROW        "Replace all throws by an error print and exit" OFF)
+option(BUILD_SFML_EXAMPLE   "Build an example using LDtkLoader with SFML." OFF)
+option(BUILD_SDL_EXAMPLE    "Build an example using LDtkLoader with SDL."  OFF)
+option(BUILD_RAYLIB_EXAMPLE "Build an example using LDtkLoader with Raylib." OFF)
```

The raylib name seems to use only lower letter (cf their website title and github project name), I think we should respect that :
 ```suggestion
option(BUILD_RAYLIB_EXAMPLE "Build an example using LDtkLoader with raylib." OFF)
```